server/server.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"context"
+	"math/rand"
+	"net"
+	"net/http"
+	"strings"
+	"sync/atomic"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/tikv/mock-tikv/server/api"
+)
+
+// Server is the mock tikv server.
+type Server struct {
+	// Configs and initial fields.
+	cfg *Config
+
+	address string
+	ctx     context.Context
+	cancel  context.CancelFunc
+
+	idAlloc  uint64
```

Comment:
Is it never used?

Reply:
Yes, it's in a very early stage and many things are just there for future.